src/pages/learn/execution.mdx

```diff
@@ -156,3 +160,18 @@ Let's take one last look at the original query to see how all these resolving fu
   }
 }
 ```
+
+As we can see, each field in the nested selection sets resolves to a scalar leaf value during execution of the query.
+
+## Next steps
+
+To recap what we've learned about execution:
+
+- Each field in a GraphQL type system will have a corresponding resolver function that provides data for the field from an existing data source
+- Execution begins at the top-level `query`, `mutation`, and `subscription` fields
```

Comment:
Should this be:

```suggestion
- Execution begins at the top-level `Query`, `Mutation`, or `Subscription` fields
```

It's not clear if the fields on the type are being referenced, or the top-level selections in the operation.